cli/tests/e2e/rules/regex-nosemgrep.yaml

```diff
@@ -2,6 +2,10 @@ rules:
   - id: detected-aws-account-id
     pattern-regex: |-
       ("|')?(AWS|aws|Aws)?_?(ACCOUNT|account|Account)_?(ID|id|Id)?("|')?\s*(:|=>|=)\s*("|')?[0-9]{12}("|')?
+    metavars:
```

Comment:
not a fan of adding yet another toplevel field. Seems like it should be under pattern-regex:

Reply:
you'd have to add yet another subsidiary key within the `pattern-regex` itself, though. like:

```
pattern-regex:
  regex: ".*"
  renames: ...
```

this seems quite disruptive to me, what do you think?